marklogic-client-api/src/main/java/com/marklogic/client/expression/GeoExpr.java

```diff
@@ -273,6 +273,9 @@ public interface GeoExpr {
   * @return  a server expression with the <a href="{@docRoot}/doc-files/types/cts_polygon.html">cts:polygon</a> server data type
   */
   public ServerExpression circlePolygon(ServerExpression circle, ServerExpression arcTolerance, ServerExpression options);
+public ServerExpression contains(ServerExpression arg1, ServerExpression arg2);
```

Comment:
The other functions have well-named args - do you know why these have unhelpful names like "arg1" and "arg2"?

Reply:
Hi Rob, could you leave your comments in this bug, 57862 - cts and geo query parameter name changed, please? Thanks.

Reply:
Done!